### ISSUE TITLE
Endpoint: http://sws.ifi.uio.no/sparql

### ISSUE THREAD

**Issue by jaygray0919** [mgskjaeveland/sgvizler#72]
Would you upgrade your endpoint to `HTTPS`?
The reason is that an `HTTPS` domain that is hosting SGvizler libraries and that accesses http://sws.ifi.uio.no/sparql is blocked by the "Mixed Content" error that states: "This request has been blocked; the content must be served over HTTPS."
We can use our hosted SGvizler libraries to request resources from https://dbpedia.org/sparql because those requests resolve to http://dbpedia.org/sparql and then process queries.

/jay gray

**Comment by mgskjaeveland** [comment]
Sgvizler does not depend on this or any other endpoint, if it does it is a bug. 
This endpoint is used for testing and for examples, and should be possible to remove.
Can you give more details to when this occurs?

**Comment by jaygray0919** [comment]
I need to re-visit this.
But, here is the issue. We are an HTTPS site. When we run a script from one of our pages against an HTTP site, the modern browsers (Mozilla, Chrome) block the request and generate the above error message.

I understand your point about Sgvizler; we do use it successfully with HTTPS endpoints.

My request had to do with http://sws.ifi.uio.no/sparql and data on that endpoint.
FYI, we have the same issue with Getty, and they are upgrading their endpoint to HTTPS.

We have developed a temporary work around: run queries from an HTTP domain that we have set up.
The problem with this is that we have to run those pages separately.
We cannot put the HTTP page into an iFrame because that also generates the Mixed Content error.
But if we have to access HTTP endpoints, at least this work around works.

So, our request is (when convenient): upgrade http://sws.ifi.uio.no to https://sws.ifi.uio.no

/jay
